<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_fast_">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_slow_">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="MouseLabel" name="label_mouse_">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mouse_move_">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MouseLabel</class>
   <extends>QLabel</extends>
   <header>mouselabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
